Pflichtenheft.tex

```diff
@@ -315,27 +313,33 @@ \section{Non-Functional Requirements}
 
 \begin{description}
 
-  \item[NF200]
+  \item[NF100]
   The rendering latency should be no longer than 2 seconds.
 
-  \item[NF300]
+  \item[NF200]
   The web UI should be viewable on modern web browsers.
 
+  \item[NF300]
+	The web UI should not crash when network connection is unstable.
+	
   \item[NF400]
-  The framework should be able to handle \glspl{data stream} from at least 500 nodes.
+  The framework should be able to handle \glspl{data stream} from at least 100 physical nodes in the network.
```

Comment:
Didn't Ankush say "hundreds of nodes"? I thought that meant physical nodes. Also, arguably, you could have one single data stream containing network traffic from hundreds of nodes.
If I'd make up some numbers, I'd say 500 nodes in 2 networks (-> 2 data streams) with say 20 different dissectors (-> 2*20 notification streams) so ca. 50 data streams.
(One actual number I found is that SIMATIC controllers can manage 256 devices)

```suggestion
  The framework should be able to handle at least 50 \glspl{data stream} and node-specific data for at least 500 physical nodes.
```

Reply:
If I remember correctly I think we "settled" on "at least 100" physical nodes, and were reminded that each node plays roles in multiple layers.

Reply:
OK, then let's keep "100 physical nodes"